test/test_wavefront_python_sdk.py

```diff
@@ -31,16 +32,34 @@ def test_get_sem_ver_value(self):
 
     def test_sanitize(self):
         """Test wavefront_sdk.common.utils.sanitize()."""
-        self.assertEqual('"hello"',
-                         sanitize('hello'))
-        self.assertEqual('"hello-world"',
-                         sanitize('hello world'))
-        self.assertEqual('"hello.world"',
-                         sanitize('hello.world'))
+        self.assertEqual('"hello"', sanitize('hello'))
```

Comment:
Did we fix this as well?
```Note that within Java source values of the form host:port are currently converted to host-port as part of the sanitization process. It'd be preferable to retain the colon in source values as they are valid characters.```
can you add unit tests as well

Reply:
Yes. I took https://github.com/wavefrontHQ/wavefront-sdk-go/commit/0755a659b61f1b14cb1feb28a13599e1d929a7bd as the reference and it is fixed by using sanitize_value() function for source rather than sanitize() function. I added some unit tests in test/test_wavefront_python_sdk.py.

Reply:
I will add more unit tests in for sanitize() function.